packages/auth-sdk/user.tsx

```diff
@@ -36,7 +37,10 @@ export function useAuthenticatedUser() {
       userKeys.user,
```

Comment:
Why are we using `z.parse()` instead of `z.safeParse()` above? Seems like we wouldn't have had the issue in https://github.com/iFixit/react-commerce/pull/2034 in the first place

Reply:
im not sure why we don't use safeParse, im assuming, like me, the writer forgot about it. However even with safeParse there would have been an issue with malformed data.

Reply:
For sure 👍🏻